package.json

```diff
@@ -23,7 +23,10 @@
     "acorn-walk": "8.2.0",
     "esbuild": "^0.17.19",
     "standard": "^17.1.0",
-    "typescript": "^5.1.3"
+    "typescript": "^5.1.3",
+    "urlpattern-polyfill": "^9.0.0",
+    "whatwg-fetch": "^3.6.17",
+    "whatwg-url": "^13.0.0"
```

Comment:
These modules are both in the `node_modules` and added as a vendor modules to the git. Was it intended?

Reply:
yes so we know what the version of the node modules we vendor

Reply:
how do we sync them? do we have some script that copies them from `node_modules`?

Reply:
no script, but we could add one like:

```sh
esbuild --bundle --format=esm node_modules/whatwg-url/index.js > api/url/url/url.js
esbuild --bundle --format=esm node_modules/whatwg-fetch/index.js > api/fetch/fetch.js
```